docs/test-fixtures.md

```diff
@@ -1,9 +1,26 @@
 <a id="top"></a>
 # Test fixtures
 
-## Defining test fixtures
+**Contents**<br>
+[Non-Templated test fixtures](#non-templated-test-fixtures)<br>
+[Templated test fixtures](#templated-test-fixtures)<br>
+[Signature-based parameterised test fixtures](#signature-based-parametrised-test-fixtures)<br>
+[Template fixtures with types specified in template type lists](#template-fixtures-with-types-specified-in-template-type-lists)<br>
 
-Although Catch allows you to group tests together as [sections within a test case](test-cases-and-sections.md), it can still be convenient, sometimes, to group them using a more traditional test fixture. Catch fully supports this too. You define the test fixture as a simple structure:
+## Non-Templated test fixtures
+
+Although Catch allows you to group tests together as [sections within a test case](test-cases-and-sections.md), it can still be convenient, sometimes, to group them using a more traditional test fixture. Catch fully supports this too with 3 different macros for non-templated test fixtures. They are: 
```

Comment:
Formatting nitpick first: new docs use linebreaks at 80 chars (well, more like 72, you know how it is).

New docs also use Catch2 to refers to the project. (The reason I don't mass update this in the old docs is that it serves as a good heuristic on how old the docs are, and how they will be written).

Reply:
I think I have addressed this now. However, I was not able to adhear to this rule with regard to tables in markdown because taking a line break would introduce a new row in the table... If you have any suggestions on how to solve this issue, I will be more than happy to do that :)

Reply:
I don't think it is possible to do with tables in github's markdown, so it's fine to keep tables without linebreaks.